hooks/accordion.php

```diff
@@ -5,8 +5,15 @@
  * behind our fixed header. See: https://wordpress.org/plugins/accordion-shortcodes/other_notes/
  */
 add_filter('shortcode_atts_accordion', function($atts) {
-    if (isset($atts['scroll']) && $atts['scroll']) {
+    if (isset($atts['scroll']) || $atts['scroll']) {
```

Comment:
changing this to an or `||` doesn't make sense to me.  It would only get to the second part of the test if `$atts['scroll']` was _not_ set, so why do `|| $atts['scroll']`? Also, would through a PHP notice if `WP_DEBUG` were on.